src/install.r

```diff
@@ -5,7 +5,11 @@ ref <- args[2]
 
 options(langserver_library = langserver_library)
 options(langserver_quiet = FALSE)
-options(repos = list(CRAN = "http://cran.rstudio.com/"))
+local({
+  current_cran <- getOption("repos")["CRAN"]
+  if (current_cran == "@CRAN@")
```

Comment:
I don't particularly like checking for the "magic" value `@CRAN@` but the docs seem to suggest it's fine:

```
Initially set from entries marked as default in the ‘repositories’ file, see
R_REPOSITORIES.  The ‘factory-fresh’ setting from ‘etc/repositories’ is
‘c(CRAN="@CRAN@")’, a value that causes some utilities to prompt for a CRAN
mirror.
```

Reply:
I am ok with that ... maybe @harmongrey has additional comments?

Reply:
Hi - sorry about the delay. I definitely think this is the most effective implementation, and good check on the `@CRAN@` value, as that could cause funky issues. Thanks for this small, but useful fix!